clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml

```diff
@@ -0,0 +1,92 @@
+      cpuPartitioningMode: AllNodes
+      clusterLabels:
+        common-du-414: true
+        group-dellr750-vse6: ""
+      clusterNetwork:
+        - cidr: 10.128.0.0/14
+          hostPrefix: 23
+        - cidr: fd01::/48
+          hostPrefix: 64
+      machineNetwork:
+        - cidr: 192.168.38.128/26
+        - cidr: 2600:52:7:300::0/64
+      serviceNetwork:
+        - 172.30.0.0/16
+        - fd02::/112
+      additionalNTPSources:
+        - clock.cars2.lab
+        - clock-v6.cars2.lab
+      # Optionally; This can be used to override the KlusterletAddonConfig that is created for this cluster:
+      # crTemplates:
+      # KlusterletAddonConfig: "KlusterletAddonConfigOverride.yaml"
+      # proxy:
+      # httpProxy: http://cars2-client.infra.cars2.lab:3128
+      # httpsProxy: http://cars2-client.infra.cars2.lab:3128
+      # noProxy: ".cars2.lab,2600:52:7:300::0/64,fd02::/112,fd01::/48,2600:52:7:300::177,2600:52:7:300::179,2600:52:7:300::180,2600:52:7:300::181"
+      nodes:
+        - hostName: "r750-2.bevo2.cars2.lab"
+          role: "master"
+          # Optionally; This can be used to configure desired BIOS setting on a host:
+          # biosConfigRef:
+          # filePath: "example-hw.profile"
+          bmcAddress: "idrac-virtualmedia://192.168.38.204/redfish/v1/Systems/System.Embedded.1"
+          bmcCredentialsName:
+            name: "r750-2-bmc-creds-secret"
+          bootMACAddress: "b4:96:91:a5:04:ac"
+          # Use UEFISecureBoot to enable secure boot
+          bootMode: "UEFI"
+          rootDeviceHints:
+            deviceName: "/dev/disk/by-path/pci-0000:65:00.0-scsi-0:2:0:0"
+          nodeNetwork:
+            interfaces:
+              - name: ens2f0
+                macAddress: "b4:96:91:a5:04:ac"
+            config:
+              interfaces:
+                - name: ens2f0
+                  type: ethernet
+                  state: up
+                  ipv4:
+                    enabled: true
+                    address:
+                      - ip: "192.168.38.151"
+                        prefix-length: 26
+                    dhcp: false
+                  ipv6:
+                    enabled: true
+                    address:
+                      - ip: "2600:52:7:300::151"
+                        prefix-length: 64
+                    autoconf: false
+                    dhcp: false
+              dns-resolver:
+                config:
+                  server:
+                    - 192.168.38.12
+                    - 2600:52:7:38::12
+              routes:
+                config:
+                  - destination: 0.0.0.0/0
+                    next-hop-address: 192.168.38.129
+                    next-hop-interface: ens2f0
+                  - destination: ::/0
+                    next-hop-address: 2600:52:7:300::1
+                    next-hop-interface: ens2f0
```

Comment:
```suggestion
                - name: eno12399np0
                  type: ethernet
                  state: up
                  ipv4:
                    enabled: true
                    address:
                      - ip: "192.168.38.151"
                        prefix-length: 26
                    dhcp: false
                  ipv6:
                    enabled: true
                    address:
                      - ip: "2600:52:7:300::151"
                        prefix-length: 64
                    autoconf: false
                    dhcp: false
                - name: ens2f0
                  type: ethernet
                  state: down
                - name: ens5f0
                  type: ethernet
                  state: down
              dns-resolver:
                config:
                  server:
                    - 192.168.38.12
                    - 2600:52:7:38::12
              routes:
                config:
                  - destination: 0.0.0.0/0
                    next-hop-address: 192.168.38.129
                    next-hop-interface: eno12399np0
                  - destination: ::/0
                    next-hop-address: 2600:52:7:300::1
                    next-hop-interface: eno12399np0
```

Reply:
switch default route to broadcom, unless i'm missing something here?  also you need to add other interfaces and set to `down` so those interfaces don't DHCP constantly (undesired).

Reply:
Not missing anything. Also added all the ports and states.